.github/workflows/test_dbt_models.yaml

```diff
@@ -22,8 +22,65 @@ jobs:
         with:
           role-to-assume: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
 
-      - name: Test models
-        run: dbt test --target "$TARGET"
+      - name: Install Python requirements
+        run: pip install -r ./.github/scripts/requirements.transform_dbt_test_results.txt
+        shell: bash
+
+      - name: Run tests
+        run: |
+          # dbt doesn't differentiate between test failures and errors, but we
+          # need to since we expect errors. Do this by capturing the output
+          # and checking its contents to look for errors
+          if output=$(dbt test --target "$TARGET" \
+                               --select \
+                                   iasworld_addn_card_not_null \
+                                   iasworld_asmt_all_class_matches_pardat_class \
```

Comment:
Note to self: Remove these lines (used to speed up testing) before merging, and replace with `tag:test_qc_*`.